backends/mock.go

```diff
@@ -0,0 +1,32 @@
+package backends
+
+// mockBackend is used for tests
+type mockBackend struct {
+	data []byte
+}
+
+func NewMockBackend() *mockBackend {
+	safeData := []byte(`-----BEGIN PGP SIGNATURE-----
```

Comment:
Please use the new format `PGP MESSAGE`. See https://github.com/leonklingele/pick/commit/e8dd22b8a222ae5f1b38d9720fd5673191bf5da3